maas/plugins/rabbitmq_status.py

```diff
@@ -98,6 +98,9 @@ def main():
 
     if r.ok:
         resp_json = r.json()  # Parse the JSON once
+        if not resp_json:
+            # Returned JSON contains no channels
+            status_err('Rabbitmq returned empty channels list')
```

Comment:
I think this should report this information in the metrics instead of exiting the script. As a general rule the scripts should only return status_err when they fail to collect data. In this case it seems that some metric should get set to 0 because there are no channels.